mod/homepage/lib/javascript/script_homepage_carousel.js.coffee

```diff
@@ -107,8 +118,32 @@ isScrolledIntoView = (elem) ->
   elemBottom = elemTop + $(elem).height();
   ((elemBottom <= docViewBottom) && (elemTop >= docViewTop));
 
-animation = (elem) ->
-  # animation CountUp.js
+runAnimation = (elem) ->
+  options = { useEasing: true, useGrouping: true, separator: ',', decimal: '.', };
+  demo = new CountUp($(elem).attr('id'),  0, parseInt($(elem).text()), 0, 3.5, options);
```

Comment:
why `demo` ?

Reply:
we need to remove the delimiter `number_with_delimiter(number, delimiter: ",")` from `hompeage_numbers.rb' ` in order make this countUp js work properly.